src/test/groovy/org/prebid/server/functional/tests/module/ortb2blocking/Ortb2BlockingSpec.groovy

```diff
@@ -268,13 +274,76 @@ class Ortb2BlockingSpec extends ModuleBaseSpec {
+        given: "Default bid request with proper ortb attribute"
+        def bidRequest = BidRequest.defaultBidRequest.tap {
+            imp.first.tap {
+                banner = Banner.getDefaultBanner().tap {
+                    battr = [PBSUtils.randomNumber]
+                }
+                video = Video.getDefaultVideo().tap {
+                    battr = [PBSUtils.randomNumber]
+                }
+                audio = Audio.getDefaultAudio().tap {
+                    battr = [PBSUtils.randomNumber]
+                }
+                ext.prebid.bidder.generic = null
+                ext.prebid.bidder.ix = Ix.default
+            }
+            imp[0].ext.prebid.bidder.generic = null
+            imp[0].ext.prebid.bidder.ix = Ix.default
+        }
+
+        and: "Account in the DB with blocking configuration"
+        def ortb2BlockingAttributeConfig = Ortb2BlockingAttributeConfig.getDefaultConfig([disallowedOrtb2Attributes], attributeName).tap {
+            enforceBlocks = true
+        }
+        def account = getAccountWithOrtb2BlockingConfig(bidRequest.accountId, [(attributeName): ortb2BlockingAttributeConfig])
+        accountDao.save(account)
+
+        and: "Default bidder response with ortb2 attributes"
+        def removeBid = getBidWithOrtb2Attribute(bidRequest.imp.first, disallowedOrtb2Attributes, attributeName).tap {
+            it.mediaType = enforceType
+        }
+        def presentBid = getBidWithOrtb2Attribute(bidRequest.imp.first, disallowedOrtb2Attributes, attributeName).tap {
+            it.mediaType = presentType
+        }
+        def bidResponse = BidResponse.getDefaultBidResponse(bidRequest).tap {
+            it.seatbid.first.bid = [removeBid, presentBid]
+        }
+        bidder.setResponse(bidRequest.id, bidResponse)
+
+        when: "PBS processes the auction request"
+        def response = pbsServiceWithEnabledOrtb2Blocking.sendAuctionRequest(bidRequest)
+
+        then: "PBS response should contain only allowed seatbid"
+        assert response.seatbid.bid.flatten().size() == 1
+        assert response.seatbid.first.bid.first.mediaType == presentType
+        assert getOrtb2Attributes(response.seatbid.first.bid.first, attributeName) == [disallowedOrtb2Attributes]*.toString()
+
+        and: "PBS response shouldn't contain any module errors"
+        assert !response?.ext?.prebid?.modules?.errors
+
+        and: "PBS response shouldn't contain any module warning"
+        assert !response?.ext?.prebid?.modules?.warnings
+
+        where:
+        disallowedOrtb2Attributes | attributeName | enforceType         | presentType
+        PBSUtils.randomNumber     | BANNER_BATTR  | BidMediaType.BANNER | BidMediaType.AUDIO
+        PBSUtils.randomNumber     | VIDEO_BATTR   | BidMediaType.VIDEO  | BidMediaType.BANNER
+        PBSUtils.randomNumber     | AUDIO_BATTR   | BidMediaType.AUDIO  | BidMediaType.VIDEO
```

Comment:
`disallowedOrtb2Attributes` could be just a variable not inside `where` the block